chart/tests/infra-mgr_test.yaml

```diff
@@ -0,0 +1,93 @@
+suite: test deploy
+values:
+  - ../values.yaml
+templates:
+  - infra-mgr.yaml
+chart:
+  version: 1.0.0
+  appVersion: 1.0.0
+tests:
+  - it: hostAliases missing
+    asserts:
+      - isKind:
+          of: Deployment
+        documentIndex: 3
+      - isNull:
+          path: spec.template.spec.hostAliases
+        documentIndex: 3
+  - it: hostAliases empty
+    set:
+      infraMgr:
+        hostAliases: []
+    asserts:
+      - isKind:
+          of: Deployment
+        documentIndex: 3
+      - isEmpty:
+          path: spec.template.spec.hostAliases
+        documentIndex: 3
+  - it: valid hostAliases
+    set:
+      infraMgr:
+        hostAliases: 
+        - ip: "127.0.0.1"
+          hostnames:
+          - "foo.local"
+    asserts:
+      - isKind:
+          of: Deployment
+        documentIndex: 3
+      - isNotEmpty:
+          path: spec.template.spec.hostAliases
+        documentIndex: 3
+      - isNotEmpty:
+          path: spec.template.spec.hostAliases[0].hostnames
+        documentIndex: 3
+      - isNotNull:
+          path: spec.template.spec.hostAliases[0].ip
+        documentIndex: 3
+      - equal:
+          path: spec.template.spec.hostAliases[0].ip
+          value: 127.0.0.1
+        documentIndex: 3
+      - equal:
+          path: spec.template.spec.hostAliases[0].hostnames[0]
+          value: "foo.local"
+        documentIndex: 3
+  - it: more than one hostAliases
```

Comment:
add test values in the previous test and remove this one, one test with a few aliases enough